corehq/apps/reports/util.py

```diff
@@ -496,15 +496,15 @@ def is_query_too_big(domain, mobile_user_and_group_slugs, request_user):
     return user_es_query.count() > USER_QUERY_LIMIT
 
 
-def send_report_download_email(title, user, link):
+def send_report_download_email(title, recipient, link):
     subject = "%s: Requested export excel data"
     body = "The export you requested for the '%s' report is ready.<br>" \
            "You can download the data at the following link: %s<br><br>" \
            "Please remember that this link will only be active for 24 hours."
 
     send_HTML_email(
```

Comment:
nit: not blocking, but we spoke a bit today about adding some more explanatory text in case this is triggered by a failed email, so the user will not be surprised to receive a link instead of the email they were expecting.